chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -60,34 +68,137 @@ The ChainIndexer [periodically runs](https://github.com/filecoin-project/lotus/b
 [Fevm]
 EthTxHashMappingLifetimeDays = 0
 
+TODO: https://github.com/filecoin-project/lotus/pull/12450#discussion_r1792743250
+> see note in #12421, this one is duplicated with a moved redirect
```

Comment:
@rvagg or @aarshkshah1992 : can you please handle this comment from @rvagg ?  I wasn't sure what to do.

Reply:
Gone now are:

* `Fevm.EthTxHashMappingLifetimeDays
* `Fevm.DisableHistoricFilterAPI`
* `Events.DisableHistoricFilterAPI`
* `Events.DatabasePath`
* `Index.EnableMsgIndex`

In https://github.com/filecoin-project/lotus/pull/12421 I've also suggested we get rid of `Fevm.Events` because it contains duplicated and previously deprecated options (we moved them to the top-level `Events`). In node/config/types.go those are held in the `DeprecatedEvents` struct which needs to go. You could say something like:

The previously deprecated `Fevm.Events` options are now also all removed.

* `Fevm.Events.DisableRealTimeFilterAPI`
* `Fevm.Events.DisableHistoricFilterAPI`
* `Fevm.Events.FilterTTL`
* `Fevm.Events.MaxFilters`
* `Fevm.Events.MaxFilterResults`
* `Fevm.Events.MaxFilterHeightRange`
* `Fevm.Events.DatabasePath`

Reply:
Addressed in https://github.com/filecoin-project/lotus/pull/12600/commits/8135376694cf753b215181dba99eb595b3fe49d1